mockfirestore/main.py

```diff
@@ -255,6 +270,13 @@ def collection(self, name: str) -> CollectionReference:
     def reset(self):
         self._data = {}
 
+    @staticmethod
```

Comment:
This isn't a static method in the real firestore library

Reply:
You’re right, but since we don’t actually rely on the state of the object itself, it is static in the Mock class.

Reply:
That's true, but I'm not sure what it having it static adds, as any code using it instead of the real firestore is going to call it as an instance method.